Hairtrigger-Bunnies/nodejs-bangazon-site
# Bobby- Feature 4

Changed:
-Added search product function in the controller
-Added new route for search products
-Adde new view to display results of searching for a product by name

Because:
-Jufe is the man

To test:
(steps to test -- include everything!)
1. Run ```npm start```
2. Login or register
3. Search for a product exactly as it is named in the database
4. The result should be the product you searched for with all relevant data associated with it
